src/components/Download/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react'
+
+import { Button, Loader } from '../Elements'
+
+import ReleaseNotes from './ReleaseNotes'
+
+import WinLogo from '../../assets/windows.svg'
+
+import './index.scss'
+
+const Download = () => {
+  const [rn, setRn] = useState('')
+  const [latestAsset, setLatestAsset] = useState({})
+
+  useEffect(() => {
+    const getData = async () => {
+      const data = await fetch('https://api.aashutosh.dev/invoicify-release')
```

Comment:
Can't we use GitHub api?

Reply:
Yeah! that's better.
I'll put on VSCode stuff until we make things public